cmd/wallet-sdk-gomobile/walleterror/error.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright Avast Software. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+// Package walleterror define error model for go mobile api.
+package walleterror
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/trustbloc/wallet-sdk/pkg/walleterror"
+)
+
+// Error represents error returned by go mobile api.
+type Error struct {
+	Code    string `json:"code"`
+	Error   string `json:"error"`
+	Details string `json:"details"`
+}
+
+// Translate translates go api errors to go mobile errors.
+func Translate(err error) error {
```

Comment:
I would suggest renaming this function to `ToMobileError` for a few reasons:

1. In the context of error messages (especially ones that may get displayed on an end-user's screen), `Translate` can be confused with language translation/localization.
2. `ToMobileError` is more precise anyway.

Reply:
done